frontend/src/views/SlideView.vue

```diff
@@ -0,0 +1,68 @@
+<!-- View for execution of a sequence -->
+<template>
+  <div class="flex w-full flex-col content-center">
+    <div class="flex flex-grow justify-center">
+      <SlideDisplayFactory
+        v-if="slide != null"
+        :slide="slide"
+        :editMode="false"
+        :key="index"
+      >
+      </SlideDisplayFactory>
+    </div>
+    <div class="flex items-center">
+      <div class="w-full mr-4">
+        <ProgressBar :percentage="percentage"> </ProgressBar>
+      </div>
+      <ButtonComponent
+        class="w-fit"
+        :loading="displaySpinner"
+        @click="nextSlideToExecute"
```

Comment:
I prefer having brackets behind functions here to indecate that they are functions